src/main/resources/yaml/es_indices_icdc.yml

```diff
@@ -622,11 +622,16 @@ Indices:
       COLLECT(DISTINCT demo.demographic_id) AS demographic_id,
       COLLECT(DISTINCT demo.additional_breed_detail) AS additional_breed_detail,
       COLLECT(DISTINCT demo.date_of_birth) AS date_of_birth,
-      COLLECT (DISTINCT (CASE bioB
-      WHEN [] THEN ['Not Applicable']
-      WHEN null THEN ['Not Applicable']
-      ELSE bioB
-      END)) AS biobank,
+      CASE 
+        COLLECT(DISTINCT(bioB))
+        WHEN []
+        THEN ['Not Applicable']
+        WHEN null
+        THEN ['Not Applicable']
```

Comment:
Should there be a case here for if `COLLECT(DISTINCT(bioB))` is equal to `s.clinical_study_designation` (or `[s.clinical_study_designation]`)? Not sure about this. Otherwise, this LGTM.